Sources/DataLoader/Channel/Channel.swift

```diff
@@ -0,0 +1,55 @@
+public actor Channel<Success: Sendable, Failure: Error>: Sendable {
```

Comment:
Could we change this from `public` to `internal` or `private`?

Reply:
I changed it to internal